File: B/Python/PC.py
from CPU import Processor
from Disk import Disk
from RAM import RAM

class PC():
    def __init__(self, cName, cPrice, dType, dCapa, dPrice, rCapa, rPrice):
        self.__processor = Processor(cName, cPrice)
        self.__disk = Disk(dType, dCapa, dPrice)
        self.__ram = RAM(rCapa, rPrice)

    def setCPU(self, i):
        self.__processor = i
    
    def getCPU(self):
        return self.__processor
    
    def setDisk(self, p):
        self.__disk = p
    
    def getDisk(self):
        return self.__disk
    
    def setRAM(self, p):
        self.__ram = p
    
    def getRAM(self):
        return self.__ram
    
    def printSpec(self):
        Processor.printCPU(self.__processor)
        Disk.printDisk(self.__disk)
        RAM.printRAM(self.__ram)
        price = Processor.getPriceCPU(self.__processor) + Disk.getPriceDisk(self.__disk) + RAM.getPriceRAM(self.__ram)
        print("Harga Total   : "+str(price))

File: B/PHP/PC.php
<?php
include "CPU.php";
include "Disk.php";
include "RAM.php";

class PC{
    private $total = 0;

    public function __construct($CPU, $RAM, $Disk){
        $this->CPU = $CPU;
        $this->RAM = $RAM;
        $this->Disk = $Disk;
    }

    public function setCPU($c){
        $this->CPU = $c;
    }

    public function getCPU(){
        return $this->CPU;
    }

    public function setRAM($r){
        $this->RAM = $r;
    }

    public function getRAM(){
        return $this->RAM;
    }
    public function setDisk($d){
        $this->Disk = $d;
    }

    public function getDisk(){
        return $this->Disk;
    }

    public function total(){
        $total = $this->CPU->getPriceCPU() + $this->RAM->getPriceRAM() + $this->Disk->getPriceDisk();
        return $total;
    }
    public function printSpec(){
        $this->CPU->printCPU();
        $this->RAM->printRAM();
        $this->Disk->printDisk();
        echo "Harga Total         : ". $this->total() ."<br>";
    }
}

?>